google/cloud/storage/blob.py

```diff
@@ -869,7 +870,12 @@ def _do_download(
             download = klass(
                 download_url, stream=file_obj, headers=headers, start=start, end=end
             )
-            response = download.consume(transport, timeout=timeout)
+            response = telemetry_wrapped_api_request(
```

Comment:
I'm not familiar with this code, but it isn't obvious to me that the blob telemetry wrappers are tested.

Reply:
added test for this